schema/rdl_schema_0.1.json

```
@@ -1219,7 +1247,8 @@
             "type": "number"
           },
           "minItems": 4,
-          "maxItems": 4
+          "maxItems": 4,
+          "uniqueItems": true
```

Per the example in https://github.com/GFDRR/rdl-standard/issues/52#issuecomment-1569450683, the items can be non-unique:

```suggestion
          "maxItems": 4
```